user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -64,46 +64,12 @@ Each row in a query corresponds to an item in the grid.
 
 You can use a grid component as a dynamic selector, i.e. a component whose selection determines behavior or data elsewhere in your dashboard or app. A common use case is controlling what is shown in another component, but selected values can also serve as input for scripts, queries, or conditional behavior.
 
+When you select an item in the grid, the selected data (*All available data* > *Components* > *Grid #* > *Selected items* > *Tables*) becomes automatically available for use in other components.
+
```

Comment:
I've removed the info that was too detailed and specific and kept it more high-level. What do you think, @SebastiaanSL?